client/src/components/Thumbnail.js

```diff
@@ -18,8 +18,10 @@ export default function Thumbnail(props) {
   const classes = useStyles(props)();
 
   return (
-    <Avatar src={props.picture || ''} className={classes.root}>
-      {props.picture ? null : <PersonIcon />}
-    </Avatar>
+    <Badge color="primary" variant="dot" invisible={props.invisible}>
+      <Avatar src={props.user.picture || ''} {...props}>
+        {props.user.picture ? null : <PersonIcon />}
+      </Avatar>
```

Comment:
```suggestion
     <Avatar src={props.picture || ''} {...props}>
        {props.picture ? null : <PersonIcon />}
      </Avatar>
```

Doing `props.user.picture` gives me the following error:
```
TypeError: Cannot read property 'picture' of undefined
```

Reply:
So, I haven't been able to figure out the issue, the code was already there and something else broke it. Idk. I just added the badge thing over it. Still figuring out this.